config/locales/client.en.yml

```diff
@@ -337,6 +337,8 @@ en:
       created: "You've bookmarked this post. %{name}"
       create_for: "Create bookmark for %{type}"
       edit_for: "Edit bookmark for %{type}"
```

Comment:
I greped codebase and looks like we can remove `create_for` and `edit_for`

Reply:
Ah yeah wasn't sure whether I should delete, I will do